src/main/java/org/jabref/logic/util/URLUtil.java

```diff
@@ -98,7 +102,12 @@ public static boolean isURL(String url) {
      * @throws MalformedURLException if the URL is malformed and cannot be converted to a {@link URL}.
      */
     public static URL create(String url) throws MalformedURLException {
-        return createUri(url).toURL();
+        if (!url.contains(PROTOCOL_SEPARATOR)) {
+            url = "http://" + url;
```

Comment:
Why not with s?

Comment needs to be added why a parsing introduces a fixed protocol!